### DISCUSSION TITLE
No side bar

### DISCUSSION THREAD

**Discussion by calixus06** [xournalpp/xournalpp#4385] [General]
Hello I have debian 11 and xournal was in synaptic . I want to work on a pdf but .. no side bar  and nothing in the edit menu. 

Thanks by advance

![pdf sans side bar](https://user-images.githubusercontent.com/30879314/200115564-9f8660d1-1068-4513-b77f-968421222aff.png)

Nico

**Comment by rolandlo** [comment]
Press F12  or use menu `View > Show Sidebar`.

**Comment by calixus06** [comment]
hello, f12 doesn't change anything and no sidebar in the show menu 


ATTENTION : changement d'adresse mail, veuillez me joindre désormais à ***@***.***




5 nov. 2022, 12:21 de ***@***.***:

**Comment by rolandlo** [comment]
The reason is simple: You installed `xournal` (which is a different project), not `xournalpp`.

**Comment by calixus06** [comment]
Thanks ! 

Where is the repository 's adress ? 

ATTENTION : changement d'adresse mail, veuillez me joindre désormais à ***@***.***




6 nov. 2022, 07:39 de ***@***.***:

**Comment by rolandlo** [comment]
You are posting this on the Github page of the repository. No need to look further. If you need more detailed installatoin instructions, see the [website](https://xournalpp.github.io/installation/).
